code/modules/mob/living/carbon/carbon.dm

```diff
@@ -376,15 +376,15 @@ var/const/GALOSHES_DONT_HELP = 8
 	if(restrained())
 		changeNext_move(CLICK_CD_BREAKOUT)
 		last_special = world.time + CLICK_CD_BREAKOUT
-		visible_message("<span class='warning'>[src] attempts to unbuckle themself!</span>", \
-					"<span class='notice'>You attempt to unbuckle yourself... (This will take around one minute and you need to stay still.)</span>")
+		visible_message("<span class='warning'>[src] attempts to uncuff themself from [buckled]!</span>", \
+					"<span class='notice'>You attempt to uncuff yourself from [buckled]... (This will take around one minute and you need to stay still.)</span>")

```

Comment:
that's an ambiguous message. It could be understood as being unbuckled and uncuffed, which is not the case.